src/Components/App/App.js

```diff
@@ -10,10 +11,12 @@ import samplePlaces from "../sampleData/samplePlaces";
 import BadURL from "../BadURL/BadURL";
 
 const App = () => {
-  const [city, setCity] = useState("Denver");
+  const [city, setCity] = useState({
+    properties: { city: "Unknown" },
```

Comment:
Just wondering if `name` in the properties would be clearer - maybe you're preparing for a json contract of some kind.